### PR TITLE
Update enterprise-grade-logging-content.md

### LINKED ISSUE #3713: Benefit to this approach vs Azure Monitor
Is there a particular scenario this guidance seeks to address? If so, that would be good to call out.

It's not clear what the benefit to this approach is compared to using existing Azure Monitor capabilities. For example, with the introduction of workspace-based App Insights resources, the assertion that 'consolidating multiple applications into a single workspace is a challenge' seems a bit out of date.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 68db9a0c-2b6a-c5aa-5fcc-428ae4cbce7b
* Version Independent ID: 68db9a0c-2b6a-c5aa-5fcc-428ae4cbce7b
* Content: [Enterprise-grade logging on Azure - Azure Reference Architectures](https://learn.microsoft.com/en-us/azure/architecture/reference-architectures/logging/enterprise-grade-logging)
* Content Source: [docs/reference-architectures/logging/enterprise-grade-logging.yml](https://github.com/microsoftdocs/architecture-center/blob/main/docs/reference-architectures/logging/enterprise-grade-logging.yml)
* Service: **architecture-center**
* Sub-service: **reference-architecture**
* GitHub Login: @yemrea
* Microsoft Alias: **yemrea**

### PR DESCRIPTION
**Pull request by yemrea**
Adding clarification statements to address concerns raised with the issue #3713.

Fixes #3713

### CODE REVIEW

**Comment by Court72** [comment]
@v-thepet 

Can you review the proposed changes? *IMPORTANT*: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

**Note**: @yemrea is listed as the author in *docs/reference-architectures/logging/enterprise-grade-logging.yml*. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: docs/reference-architectures/logging/enterprise-grade-logging-content.md [hunk 1]
```diff
@@ -22,8 +24,9 @@ This reference architecture describes how to achieve enterprise-grade logging on
 
 ### Alternatives
 
-- The Application Insights and Log Analytics features of [Azure Monitor](/azure/azure-monitor) can do end-to-end tracing and troubleshooting. These features use Azure Data Explorer behind the scenes. Azure Monitor has capabilities and benefits that are similar to the current architecture, but consolidating multiple applications into a single workspace is a challenge in Azure Monitor.
-
+- The Application Insights and Log Analytics features of [Azure Monitor](/azure/azure-monitor) can do end-to-end tracing and troubleshooting. These features use Azure Data Explorer behind the scenes. Azure Monitor has capabilities and benefits that are similar to the current architecture, but consolidating multiple applications into a single workspace is a challenge in Azure Monitor. [Workspace-based Application Insights](/azure/azure-monitor/app/create-workspace-resource) provides a capability to centralize logs from Application Insights into a common Log Analytics workspace. Azure Monitor allows custom logs to be ingested and data format to be extended for these custom logs. There are two different pitfalls with this alternate:
```
Comment (ckittel) [inline docs/reference-architectures/logging/enterprise-grade-logging-content.md:27]:
```suggestion
- The Application Insights and Log Analytics features of [Azure Monitor](/azure/azure-monitor) can do end-to-end tracing and troubleshooting. These features use Azure Data Explorer behind the scenes. Azure Monitor has capabilities and benefits that are similar to the current architecture, but consolidating multiple applications into a single workspace is a challenge in Azure Monitor. [Workspace-based Application Insights](/azure/azure-monitor/app/create-workspace-resource) provides a capability to centralize logs from Application Insights into a common Log Analytics workspace. Azure Monitor allows custom logs to be ingested and data format to be extended for these custom logs. There are two different tradeoffs with this alternative:
```

File: docs/reference-architectures/logging/enterprise-grade-logging-content.md [hunk 2]
```diff
@@ -22,8 +24,9 @@ This reference architecture describes how to achieve enterprise-grade logging on
...
+  - Custom logs will be ingested to different tables than original tables. `trackEvent()` for custom logs will be stored in `customEvents` table whereas original events will be stored in `Events` table. 
+  - Developers can extend table schema for custom tables as needed which is a feature of Azure Monitor. Without a strict governance model, table schema can become complicated over time. With the help this reference architecture, you can introduce your own governance model whereas you can still benefit from [Workspace-based Application Insights](/azure/azure-monitor/app/create-workspace-resource) capabilities by running cross cluster queries.
```
Comment (ckittel) [inline docs/reference-architectures/logging/enterprise-grade-logging-content.md:29]:
```suggestion
  - Developers can extend table schema for custom tables as needed which is a feature of Azure Monitor. Without a strict governance model, table schema can become complicated over time. With the guidance from this architecture, you can introduce your own governance model and still benefit from [Workspace-based Application Insights](/azure/azure-monitor/app/create-workspace-resource) capabilities by running cross-cluster queries.
```

**Review by ckittel — CHANGES_REQUESTED** [review]
Looks like you covered some of the feedback from @ejhenry on MicrosoftDocs/architecture-center#3713 - thanks! But can you maybe touch on the core question presented by @ejhenry there, which is -- what scenario is this solution this beneficial for? Meaning, what's the "decision tree" branch that you hit when you decide to go down this path?  Should EVERY customer go down this path, or only some and if it's some -- what was that trigger that indicates that people like them should do this?

@ejhenry -- would you like to review this proposal to see if it covers your feedback?  I don't wish to speak for you.

Also, @yemrea, as an FTE, we encourage you to open PRs on our internal repo which gives you a richer experience, instead of in the public repo here.  Ping me on teams if you'd like to learn more.

**Comment by yemrea** [comment]
I believe, this should be applicable for every enterprise customer. I guess, it was not clear for Eric that we are talking about custom application logs. My changes are intended to highlight these are application logs.

**Review by ejhenry — DISMISSED** [review]
Looks good to me. Thanks for adding the additional clarification! 👍

**Review by ckittel — APPROVED** [review]
Thanks @yemrea and @ejhenry!

**Comment by ckittel** [comment]
#sign-off

**Review by EdPrice-MSFT — APPROVED** [review]
Looks good!

**Comment by EdPrice-MSFT** [comment]
#Sign-Off

**Pull request merged**